test/new_relic/agent/method_tracer_test.rb

```diff
@@ -242,6 +249,7 @@ def test_code_level_metrics_for_an_anonymous_method
       cls.new.instance_method
 
       attributes = txn.segments.last.code_attributes
+
       assert_equal __FILE__, attributes['code.filepath']
       assert_equal 'instance_method', attributes['code.function']
       assert_equal 220, attributes['code.lineno']
```

Comment:
The failing tests are because of the new lines added before line 220, moving the location of the codes line number down to 226, so I think this number just needs to be updated.

Reply:
Thank you, Tanna!